src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent, ReactNode } from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+import "lbh-frontend/lbh/components/lbh-error-message/_error-message.scss";
+
+export type ErrorMessageProps = {
+  id?: string;
+  className?: string;
+  visuallyHiddenText: string;
+  children?: ReactNode;
+};
+
+export const ErrorMessage: FunctionComponent<ErrorMessageProps> & {
+  defaultProps: Partial<ErrorMessageProps>;
+} = ({ id, className, visuallyHiddenText, children }) => {
+  const messageClassName = classnames(
+    "govuk-error-message",
+    "lbh-error-message",
+    className
+  );
+  return (
+    <span id={id} className={messageClassName}>
+      {visuallyHiddenText && (
+        <span className="govuk-visually-hidden">{visuallyHiddenText}:</span>
+      )}
+      {children}
+    </span>
+  );
+};
+
+ErrorMessage.propTypes = {
+  id: PropTypes.string,
+  className: PropTypes.string,
+  visuallyHiddenText: PropTypes.string.isRequired,
```

Comment:
```suggestion
  visuallyHiddenText: PropTypes.string,
```

I don't think it's required as a prop type, because the prop types determine what's required as input.